modules/ztp-image-based-upgrade-without-backup.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrading-without-backup_{context}"]
+= Upgrading the {sno} clusters with {lcao} without backup
+
+When you have generated the seed image and have completed the `Prep` stage, you can begin upgrading the target clusters.
+During the upgrade process, the OADP Operator creates a backup of the cluster, then {lcao} reimages it.
+
+[IMPORTANT]
+====
+If the upgrade fails or stops, you must roll back the upgrade.
```

Comment:
There will be two types of rollbacks: 
1) Automatic rollbacks where the system initiates the rollback due to a fatal error
2) User initiated rollback where the admin can rollback the system in the event issues are found after the upgrade has completed